Cargo.toml

```diff
@@ -2,13 +2,37 @@
 name = "pyn5"
 version = "1.0.0"
 authors = ["pattonw <pattonw@hhmi.org>"]
+description = "Python wrapper around rust-n5."
+readme = "README.rst"
+license = "MIT"
+repository = "https://github.com/pattonw/rust-pyn5"
+homepage = "https://github.com/pattonw/rust-pyn5"
+documentation = "https://rust-pyn5.readthedocs.io"
+
+[package.metadata.maturin]
+classifier = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Rust",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+]
+requires-dist = ["numpy", "h5py_like>=0.6.0"]
+requires-python = ">=3.6"
+
+[badges]
+travis-ci = { repository = "pattonw/rust-pyn5" }
 
 [lib]
 name = "pyn5"
 crate-type = ["cdylib"]
 
 [dependencies]
 pyo3 = { version = "0.7", features = ["extension-module"] }
-n5 = { version = "0.4", default-features = false, features = ["filesystem", "bzip", "gzip", "lz", "xz", "use_ndarray"]}
+# liblzma breaks manylinux compatibility
+# n5 = { version = "0.4", default-features = false, features = ["filesystem", "bzip", "gzip", "lz", "xz", "use_ndarray"]}
```

Comment:
I'm a little confused here since I'm pretty sure the first release was built on manylinux by the cibuildwheels tool, and seemed to work with both "lz" and "xz" features. I don't remember if I had to do something special to get it to work though.

Just a cleanliness/tracking note, would it be better to write up a quick issue for this rather than keeping the old code as a comment? I think it would probably be fine to leave it as is if it were only touched on here, but it looks like there's a couple files that will need to be touched to re-add these compression formats, so it might be nice to keep track in an issue, and then we can remove the comments.

Reply:
> the first release was built on manylinux by the cibuildwheels tool, and seemed to work with both "lz" and "xz" features.

Yeah, I thought about that too. `maturin` contains a partial re-implementation of auditwheel so it's possible it's doing different checks.

I can certainly remove these lines and add an issue (I already raised an issue on rust-n5 to consider pure-rust lz/xz implementations).

Reply:
To add to this: I would prefer to keep the commented-out code in `conftest.py` and the compression type enum, so that we're clear on what it will look like when they come back and hopefully it'll just be temporary.

Reply:
sounds good, I'm cool with keeping the comments, I just don't know how long these sort of things tend to take, so I thought having an issue would be easier to remember to resolve than commented code.